app.amsterdam.json

```diff
@@ -16,12 +17,12 @@
     "fetchQuestionsFromBackend": false,
     "mapFilter24Hours": true,
     "reporterMailHandledNegativeContactEnabled": true,
+    "showMainCategories": true,
+    "showStandardTextAdminV2": true,
```

Comment:
Geen showStandardTextAdminV1 in app.amsterdam.json? Willen we hier gelijk live op acceptatie mee?

Reply:
Default is true, so not necessary